packages/web/components/control/menu-dropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import { FunctionComponent } from "react";
+import classNames from "classnames";
+import { MenuSelectProps } from "./types";
+
+interface Props extends MenuSelectProps {
+  isOpen: boolean;
+}
+
+export const MenuDropdown: FunctionComponent<Props> = ({
+  options,
+  selectedOptionId,
+  onSelect,
+  isOpen,
+}) => (
+  <div
+    className={classNames(
+      "absolute flex flex-col w-36 bg-card rounded-[9px] border border-white-faint select-none",
+      {
+        hidden: !isOpen,
+      }
+    )}
+  >
+    {options.map(({ id, display }, index) => (
+      <span
+        className={classNames(
+          "px-2 cursor-pointer w-full text-ellipsis overflow-hidden hover:text-clip hover:bg-white-faint",
+          {
+            "bg-white-faint text-white-full": id === selectedOptionId,
+            "text-iconDefault": id !== selectedOptionId,
+            "rounded-t-[8px]": index === 0,
```

Comment:
It would better use `rem` rather than `px` to manage size proportionally. Also In here, it would be better to use a predefined class in the tailwind like `rounded-t-lg`.

Reply:
Actually this one is weird, the container needs to be 9px border radius, and the inner elements need to be 1 px less since they are 1px smaller with the container 1px border. Otherwise, we're left with a small gap on the corners of the inner elements.

Reply:
See:
![image](https://user-images.githubusercontent.com/4606373/152445286-c213afe1-0958-4178-a0bd-a5b552a5f960.png)

Reply:
@victortokaito pointed this out. That screenshot is with tailwind `rounded-lg` I believe

Reply:
@jonator I can't find the inner elements' border radius need to be 8px on figma. It looks to be auto layout.

Reply:
@victortokaito Speaking of '9px', I prefer width, padding, border-radius etc's value(px) to be even number. In tailwind, There are predefined classes like `rounded-lg` and their value is almost even. If the value is odd number, developer should make new class and calculating `px` to `rem` is little bit tricky and weird(9px -> 0.5625rem).

@jonator How think about this?

Reply:
I added the `lginset` class. LMK if this is fitting for using tailwind.